pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,86 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
```

Comment:
this cuts across a lot of object types... why does a registry admin need to create projects?